view/frontend/web/js/view/payment/method-renderer/omise-offline-tesco-method.js

```diff
@@ -85,7 +76,9 @@ define(
                 self.getPlaceOrderDeferredObject()
                     .fail(
                         function(response) {
-                            self.handlePaymentError(response);
```

Comment:
handling payment error is in one place only, when dealing with tesco payment, that is why I moved it from function and added as was previously.